setup.py

```diff
@@ -2,12 +2,13 @@
 
 install_requires = [
     "huggingface_hub",
-    "cloudpickle==1.6"
+    "cloudpickle==1.6",
+    "pickle5"
 ]
```

Comment:
I would suggest adding some extras for code style with black, isort, and flake8. This helps follow cleaner code.

See https://github.com/huggingface/huggingface_hub/blob/main/setup.py#L38-L42 as a pointer. If you add that and you do `make style` and `make quality` it will do all of these runs.